pages/docs/manual/latest/pipe-last.mdx

```diff
@@ -0,0 +1,22 @@
+# Pipe Last
+
+[`|>`](https://reasonml.github.io/api/Pervasives.html#VAL(|%3E)) is similar to [pipe first (`->`)](./pipe-first) but it sends the left side value to the last argument of the right side. `c |> a(b)` becomes `a(b, c)`.
+
+Note that pipe last may sometimes have a runtime cost due to currying.
```

Comment:
- The cases where `|>` produces a runtime function call are few and far between, thanks to lots of compiler optimization, so highlighting that as a difference is not strictly wrong, but it's a bit misleading
- `_` is a language feature unrelated to pipes, and it works fine with pipe last, so this is inaccurate
- `getExn` isn't a pattern that should be encouraged... maybe something like `map` here instead?

Reply:
@mlms13 see my review here https://github.com/reason-association/reasonml.org/pull/178#pullrequestreview-405221791

Reply:
Ah, inline review. Good call :)

Reply:
Thanks I will update my PR based on the comments.

Reply:
Thanks for the feedback!

@mlms13 https://github.com/reason-association/reasonml.org/pull/178/commits/d40265717ca1a83ddf1c65010e72ab81f1ea9fe7

^ I used `isSome` instead of the suggested `map` because `map` takes 2 arguments and the 2nd argument is a transforming function. Let me know if this seems ok!